### PR TITLE
change drop-role-handling and add support for set-account-alias

### PR DESCRIPTION
**Pull request by tuxlife**
* add support for iam role "only_unused" and drop role only if this a session without credentials profile
* set account alias if the alias not set (needed for new AWS Organizations accounts)

### CODE REVIEW

File: sevenseconds/config/__init__.py [hunk 1]
```diff
@@ -67,9 +67,11 @@ def configure_account(session_data: AccountData, trusted_addresses: set):
     boto3.DEFAULT_SESSION = None
     # account_id = get_account_id(session['account'])
     # info('Account ID is {}'.format(account_id))
-
+    isProfileSession = 'profile_name' in session_data.session
```
Comment (hjacobs) [inline sevenseconds/config/__init__.py]: please change to Python code style (`is_profile_session`)

**Comment by hjacobs** [comment]
@tuxlife can you describe the "why"/purpose? I don't understand the intention yet.

**Comment by tuxlife** [comment]
The original purpose was, that we remove "helper" roles, if the current session doesn't use the role.

But i didn't found a easy way to get the role-name for the current access-key-id.

**Comment by hjacobs** [comment]
the purpose of this PR is to allow using the organization API which creates an initial role which should be dropped later (but needs to be used the first time so we check that the current session is not using it before dropping)

**Comment by hjacobs** [comment]
:+1:

**Comment by tuxlife** [comment]
:+1:

**Pull request merged**